tests/tracking/test_model_registry.py

```diff
@@ -64,9 +64,7 @@ def server_urls():
     Clean up all servers created for testing in `pytest_generate_tests`
     """
     yield
-    for server_url, process in BACKEND_URI_TO_SERVER_URL_AND_PROC.values():
```

Comment:
Let's replace print functions in this file with logging methods.

Reply:
Done! I put INFO level logs